### PR TITLE
pymodule for time module

### PR DESCRIPTION
**Pull request by youknowone**
cc @Snowapril 

review link: https://github.com/RustPython/RustPython/pull/3009/files?w=1

### CODE REVIEW

**Comment by Snowapril** [comment]
This is extremely clear than my version! 😊

**Review by Snowapril — APPROVED** [review]
This is extremely clear than my version! 😊

File: vm/src/stdlib/time.rs [hunk 1]
```diff
@@ -79,409 +29,515 @@ pub fn get_time() -> f64 {
...
+        target_os = "linux",
+        target_os = "illumos",
+        target_os = "netbsd",
+        target_os = "solaris",
+        target_os = "openbsd",
+        target_os = "redox",
+        all(target_arch = "wasm32", not(target_os = "unknown"))
+    )))]
+    fn get_process_time(vm: &VirtualMachine) -> PyResult<std::time::Duration> {
+        Err(vm.new_not_implemented_error("thread time unsupported in this system".to_owned()))
```
Comment (Snowapril) [inline vm/src/stdlib/time.rs:265]:
```suggestion
        Err(vm.new_not_implemented_error("process time unsupported in this system".to_owned()))
```
Maybe this is typo :)

**Review by DimitrisJim — APPROVED** [review]
lgtm (though had to look through each commit for a better view of the changes!)

**Pull request merged**